content/shmem_query_thread.tex

```diff
@@ -0,0 +1,36 @@
+\apisummary{
+Returns the level of thread support provided by the library.}
+
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void shmem_query_thread(int *provided);
+\end{Csynopsis}
+
+\begin{apiarguments}
+\apiargument{OUT}{provided}{The thread level support provided by the \openshmem{} implementation.}
+\end{apiarguments}
+
+
+\apidescription{
+The \FUNC{shmem\_query\_thread} call returns the level of thread support
+currently being provided. The value returned will be same as \VAR{provided}
+returned in the \FUNC{shmem\_init\_thread}, if the \openshmem{} library was
+initialized by \FUNC{shmem\_init\_thread}. If the library was initialized by
+\FUNC{shmem\_init}, the implementation can choose to provide one of the defined 
+thread levels, and \FUNC{shmem\_query\_thread} returns this thread level.
```

Comment:
Suggest rewording to: If the library was initialized by shmem_init, the thread level returned by shmem_query_thread is implementation-defined.